frontend/src/static/js/v_authorship.js

```diff
@@ -32,6 +32,7 @@ window.vAuthorship = {
     return {
       isLoaded: false,
       files: [],
+      totalLineCnt: '',
```

Comment:
I guess you are following the existing convention; just note that we should use full words as much as possible i.e., `Cnt` -> `Count`

Reply:
Yes, I was following the existing naming convention. I've renamed ```totalLineCnt``` to ```totalLineCount```